stringutils.go

```diff
@@ -70,3 +73,29 @@ func hashsum(s string, hasher hash.Hash) string {
 	hasher.Write([]byte(s))
 	return hex.EncodeToString(hasher.Sum(nil))
 }
+
+func foundInHIBP(s string) (int, error) {
```

Comment:
Looking at what this function does, maybe it's enough to simply return either `nil` or an actual error? After all we don't really handle the returned integer index, we're merely handling the following outcomes:

1. The hash is unknown
2. The hash is known
3. An error occurred talking to the API